hardware_interface_testing/test/test_components/test_sensor.cpp

```diff
@@ -30,7 +30,7 @@ class TestSensor : public SensorInterface
       return CallbackReturn::ERROR;
     }
     // can only give feedback state for velocity
-    if (info_.sensors[0].state_interfaces.size() != 1)
+    if (info_.sensors[0].state_interfaces.size() == 2)
```

Comment:
@destogl  Is this change needed?

Reply:
yes. I have adjusted simulating error and other things here. I am using the number of interfaces as “switch” between different behaviors with interfaces.

Reply:
Understood. Thanks for the explanation.